packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js

```diff
@@ -28,6 +29,8 @@ export default class BillingPaymentMethodAddCtrl {
     this.loading = {
       redirecting: false,
     };
+
+    this.hostname = new URL(CORE_MANAGER_URLS.dedicated).hostname;
```

Comment:
i would simply use window.location.hostname

Reply:
Hmm I did this, but for testing purpose, I add the constant to have the production correct hostname (and not staging link)

Reply:
Constant has been replaced by `window.location.hostname`